unlock-app/src/stories/interface/PaginationPageNumbers.stories.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Provider } from 'react-redux'
+import PageNumbers from '../../components/interface/pagination/PageNumbers'
+import createUnlockStore from '../../createUnlockStore'
+
+const store = createUnlockStore({
+  transactions: {
+    deployedid: {
+      status: 'mined',
+      confirmations: 24,
+    },
+    confirmingid: {
+      status: 'mined',
+      confirmations: 4,
+    },
+    submittedid: {
+      status: 'submitted',
+      confirmations: 0,
+    },
+    withdrawalconfirmingid: {
+      status: 'mined',
+      confirmations: 2,
+      withdrawal: 'withdrawalconfirmingaddress',
+    },
+    withdrawalsubmittedid: {
+      status: 'submitted',
+      confirmations: 0,
+      withdrawal: 'withdrawalsubmittedaddress',
+    },
+  },
+  keys: {
+    keyid: {
+      transaction: '0x23749328748932748932473298473289473298',
+      lock: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e',
+      expiration: Math.floor(new Date().getTime() / 1000) + 86400 * 30, // 30 days from right now
```

Comment:
This needs to be changed, the snapshot tests will fail every new day :). You can see in the snapshots that "Feb 7, 2019" exists in the markup for every key, and that today, the snapshots failed and had to be updated to "Feb 8, 2019" Tomorrow they will need another update. Just hard-code a date here, it's fine for storybook purposes